.mergify.yml

```diff
@@ -0,0 +1,20 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>=1"
+      - label!=not-ready
+
+pull_request_rules:
+  - name: Automatic merge on approval & CI
+    conditions:
+      - "#approved-reviews-by>=1"
```

Comment:
It is good to check that there is at least one approval, but I saw there is also a way to check that there are no pending review. Can you add this also ?

Reply:
done with `"#review-requested=0"` condition